test/blackbox-tests/test-cases/coq/extract/run.t

```diff
@@ -16,7 +16,7 @@
   $ cat >dune <<EOF
   > (coq.extract
   >  (prelude extract)
-  >  (extracted_modules Datatypes Extract))
+  >  (extracted_modules datatypes extract))
```

Comment:
So indeed Coq's convention is respect the capitalization of the original module, AFAICT OCaml doesn't care about this, so why not use here `Datatypes extract` ?

Reply:
I followed your advice and it's now `Datatypes extracts`. Indeed we should just treat this as an opaque list of strings.